src/test/ui/async-await/await-into-future.rs

```diff
@@ -0,0 +1,23 @@
+// check-pass
+
+// edition:2018
+
+struct AwaitMe;
+
+impl IntoFuture for AwaitMe {
```

Comment:
Doesn't this need to be imported? Also, I'd expect some stability annotations. Perhaps the tests will fail, though. Otherwise, seems like we have a "hole" :)